tox.ini

```diff
@@ -26,19 +24,21 @@ extras =
 commands =
   ruff format --check qiskit_addon_sqd/ docs/ test/
   ruff check qiskit_addon_sqd/ docs/ test/
+  ruff check --preview --select CPY001 --exclude "*.ipynb" qiskit_addon_sqd/ test/
   nbqa ruff docs/
+  pydocstyle qiskit_addon_sqd/
   mypy qiskit_addon_sqd/
-  #reno lint
   pylint -rn qiskit_addon_sqd/ test/
   nbqa pylint -rn docs/
-  pydocstyle qiskit_addon_sqd/
+  typos
+  reno lint
 
-[testenv:{,py-,py3-,py39-,py310-,py311-,py312-}notebook]
+[testenv:{,py-,py3-,py310-,py311-,py312-}notebook]
 extras =
   nbtest
   notebook-dependencies
 commands =
-  pytest --nbmake --nbmake-timeout=3000 {posargs} docs/tutorials/
+  pytest --nbmake --nbmake-timeout=3000 {posargs} docs/
 
 [testenv:coverage]
```

Comment:
I did not align this tox target but it differs a bit from the one in `qiskit-addon-utils`: https://github.com/Qiskit/qiskit-addon-utils/blob/aa32a44a416c3c9e4008164c87730026f7924dee/tox.ini#L51-L59

Not sure if you had a reason for relying on the more manual `coverage3` calls

Reply:
The unit tests are VERY sparse right now, so I did not put much thought into coverage, nor do I run it in CI. Was planning on giving this a closer look in the coming weeks

Reply:
Ive ignored all the slow notebooks + dice_solver notebook and now run nbmake on all of docs/ as you have here.